test/support/codeflare.go

```diff
@@ -30,8 +33,16 @@ const (
 	CodeFlareTestPyTorchImage = "CODEFLARE_TEST_PYTORCH_IMAGE"
 
 	// The testing output directory, to write output files into.
-
 	CodeFlareTestOutputDir = "CODEFLARE_TEST_OUTPUT_DIR"
+
```

Comment:
I realise that file should probably be called `environment.go`, WDYT?

Reply:
Updated